spec/observables/combineLatest-spec.js

```diff
@@ -28,4 +28,397 @@ describe('Observable.combineLatest', function () {
       done();
     });
   });
```

Comment:
Personal preference is code to be strictly guarded even there is some of duplication may occurs. Course it'd be better if it's possible to have common fixture setup to reduce duplication..